### PR TITLE
edited docs/getting_started.md

### PR DESCRIPTION
**Pull request by kwalrath**
An attempt to make getting started a little easier.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -1,58 +1,122 @@
...
 
-## Available commands
 
-In addition to `serve` you can use:
+## Using `build_runner` as a development server
 
-- `build` - run a single build and exit. This is most useful if your build also
-  generates outputs to your source directory. With `--output <dirname>` this
+1. Edit your package's **pubspec.yaml** file,
+   removing all transformers and
```
Comment (natebosch) [inline docs/getting_started.md]: removing transformers is optional - people can try it out without breaking their existing `pub serve` workflow.

File: docs/getting_started.md [hunk 2]
```diff
@@ -1,58 +1,122 @@
...
-- `watch` - like `build` but reruns after file changes. With
+
+- **watch:** Like `build` but reruns after file changes. With
   `--output <dirname>` the merged output directory will be kept up to date with
   changes. This can be used to keep the outputs updated for use with another
   filed-based development server.
-- `test` - Creates an output directory and runs `pub run test` within it.
 
-## Switching to Dart2Js
+- **test:** Creates an output directory and runs `pub run test` within it.
```
Comment (natebosch) [inline docs/getting_started.md]: Should we call out that `build_test` needs to be a dev dependency for this to work? The latest version of the command will check for it and warn you - but that might be a surprise

File: docs/getting_started.md [hunk 3]
```diff
@@ -1,58 +1,122 @@
...
 targets:
   my_package_name:
     builders:
-      dart_web_compilers|entrypiont:
+      dart_web_compilers|entrypoint:
         options:
           compiler: dart2js
 ```
 
+<!--PENDING: Why does this show a `builders` line? Is that necessary for
```
Comment (natebosch) [inline docs/getting_started.md]: Is it is necessary. The `dart_web_compiler|entrypoint` is a "builder" just like angular or any other builder - there isn't anything special about compilers in build_runner like there was with pub.

File: docs/getting_started.md [hunk 4]
```diff
@@ -1,58 +1,122 @@
...
+with `pub run build_runner`, because it will be kept up to date with changes
+in the build packages. If your pubspec lists builders, switch to a version of
+the builder-containing package that has a `build.yaml` file.
+
+
+## Troubleshooting
+
+<!-- summarize here. -->
+
+<!-- add something about symptoms if you don't remove transformers -->
```
Comment (natebosch) [inline docs/getting_started.md]: There shouldn't be any symptoms for including or removing the config.

File: docs/getting_started.md (hunk 2 above)
Comment (jakemac53) [reply docs/getting_started.md]: ya we should add that imo

File: docs/getting_started.md [hunk 5]
```diff
@@ -1,58 +1,122 @@
...
 Older versions of `build_runner` were designed to run with manually written
 build scripts referencing the Builders available in the local package or in
 dependencies. This pattern can still be used when customization is needed
-outside of `build.yaml`, but we recommend most users use the generated build
-script with `pub run build_runner` since it will be kept up to date with changes
-in the build packages. Check with the packages that provide Builders you are
-using for versions published with a `build.yaml` file.
+outside of `build.yaml`, but we recommend using the generated build script
+with `pub run build_runner`, because it will be kept up to date with changes
+in the build packages. If your pubspec lists builders, switch to a version of
```
Comment (jakemac53) [inline docs/getting_started.md]: s/builders/transformers

although this doesn't seem relevant to this section (its duplicated from the above section where it is relevant?)

File: docs/getting_started.md (hunk 4 above)
Comment (jakemac53) [reply docs/getting_started.md]: We could add a warning if we still see a transformers section or something, idk

File: docs/getting_started.md (hunk 2 above)
Comment (kwalrath) [reply docs/getting_started.md]: Done.
